corehq/ex-submodules/auditcare/utils/__init__.py

```diff
@@ -1,31 +1,28 @@
 #source, from django-tracking
 
+from threading import local
```

Comment:
The previous version of this was enclosed in a try/except block -- what makes this safe now?

Reply:
`threading.local` is part of the standard library [since Python 2.4](https://docs.python.org/2/library/threading.html#threading.local). It seems to me that the previous code is misguided on modern Python. I'm actually not even sure what situation that try/except was meant to handle. It seems to me that it would never fail unless you were running on a very old or weird/patched version of Python?